tests/core_tests/config_test.py

```diff
@@ -55,7 +55,7 @@ def test_immutability(self):
         test_config_1 = Config(self.param_pos, self.ic_pos)
         test_config_2 = Config(self.param_all, self.ic_all)
         with self.assertRaises(MutationException):
-            test_config_1.init_cond = State(
+            test_config_1.init_cond = StateTime(
                 {"throw": "error"}
```

Comment:
A `StateTime` cant be init'ed with a dict like this. You'll want to use `StateTime.from_dict(...)`, but then also need to change `"error"` to an int, float, or bool